### PR TITLE
Add a tmpfiles.d lens

### PR DESCRIPTION
**Pull request by roidelapluie**

### CODE REVIEW

File: lenses/tmpfiles.aug [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+                   [ label "gid" . store field ] . ( sep_spc .
+                   [ label "age" . store age ] . ( sep_spc .
+                   [ label "argument" . store argument ] )? )? )? )? )? .
+                     Util.comment_or_eol ]
+
+  let lns = ( empty | comment | record ) *
+
+  let filter = (incl "/etc/tmpfiles.d/*.conf")
+             . (incl "/usr/lib/tmpfiles.d/*.conf")
+             . (incl "/run/tmpfiles.d/*.conf")
```
Comment (raphink) [inline lenses/tmpfiles.aug]: brackets are not required here

**Comment by raphink** [comment]
As stated on IRC, could you add some docs as in keepalived.aug?

Also, the build will most likely fail since you didn't add the lens to `tests/Makefile.am`.

**Comment by roidelapluie** [comment]
Build is green and code is commented.

**Comment by raphink** [comment]
If you add NaturalDocs comments before every "let" definition (ideally also in the test file), it will allow to automatically generate docs on the site, so that would be great.

Also, can you add the files to doc/naturaldocs/conf/lenses/Menu.txt as well?

**Comment by roidelapluie** [comment]
@raphink I have addressed your comments.

**Comment by raphink** [comment]
LGTM, I'll merge.

**Comment by raphink** [comment]
Merged as aeb9f38

**Pull request closed without merging**